GoogleForCreators/web-stories-wp
# Story Poster: Use dimensions array instead of size name

## Context



## Summary
Another solution to #12354. If you pass an array of dimensions instead of passing size name it bypasses the content width check. [See](https://github.com/WordPress/wordpress-develop/blob/5eddc8f18c97d2eb909d0f70f4e20732398a7bda/src/wp-includes/media.php#L69-L71)

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

0. Activate Twenty Twenty theme
0. Create a new story and add a few pages
0. Add a poster image by uploading a new image that needs to be cropped & crop it
0. Save the story
0. Reload the page
0. View checklist, see no incorrect poster dimensions.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?


## Checklist




---



Fixes #12354
